todocli/test/test_datetime_parser.py

```diff
@@ -0,0 +1,106 @@
+import unittest
+from datetime import datetime, timedelta
+
+from todocli.datetime_parser import (
+    parse_datetime,
+    add_day_if_past,
+    ErrorParsingTime,
+    TimeExpressionNotRecognized,
+)
+
+
+class TestDatetimeParser(unittest.TestCase):
+    def test_sadd_day_if_datetime_is_in_past(self):
```

Comment:
I think `sadd` is a typo? If not, ignore this comment.

Reply:
fixed